### PR TITLE
Feature/how we help

### PR DESCRIPTION
**Pull request by stuaxo**
How we help.
Site logo also sneaked in here too

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -88,6 +88,12 @@
     'django.contrib.staticfiles',
 ]
 
+try:
+    import django_extensions
+    INSTALLED_APPS.append('django_extensions')
```
Comment (sdonk) [inline config/settings.py]: This smells of "we need a different settings file" to me :)

Can you please move `INSTALLED_APPS.appen()` to the `else` clause? Only the line causing the exception should be in the `try` block

File: home/models.py [hunk 2]
```diff
@@ -17,10 +20,18 @@ class HomePage(Page):
         related_name='+'
     )
 
+    how_we_help = StreamField([
+        ('items', StructBlock([
+            ('icon', ImageChooserBlock()),
+            ('text', CharBlock()),
+        ]))
+    ], blank=True)
```
Comment (sdonk) [inline home/models.py]: this is not really readable, can you please match the indentation on opening and closing brackets?

File: invest/templates/home/home_page.html [hunk 3]
```diff
@@ -10,4 +10,5 @@
   {% include "includes/heading_hero_tall.html" %}
   {% include "includes/image_cards.html" with cards=sector_cards title="Discover UK Industries" %}
   {% include "includes/setup_guide_cards.html" with cards=setup_guide_cards title="UK Setup Guide" %}
+  {% include "includes/how_we_help_cards.html" with cards=page.how_we_help title="How We Help" header="The Department for International Trade (DIT) helps overseas companies relocate and expand in the UK. Based in British embassies, high commissions and consulates in cities around the world, our investment services team can:" %}
 {% endblock %}
```
Comment (sdonk) [inline invest/templates/home/home_page.html]: new line?

File: invest/templates/includes/how_we_help_cards.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        <div class="card-deck-wrapper">
+          <div class="card-deck">
+          {% for card in cards %}
+            {% include "includes/how_we_help_card.html" %}
+          {% endfor %}
+          </div>
+        </div>
+      </div>
+  </div>
+</section>
```
Comment (sdonk) [inline invest/templates/includes/how_we_help_cards.html]: new line?

File: config/settings.py (hunk 1 above)
Comment (stuaxo) [reply config/settings.py]: :+1:

I couldn't work out how to get the ```make test``` to not moan about ```django_extensions``` being imported and not used.

File: home/models.py (hunk 2 above)
Comment (stuaxo) [reply home/models.py]: Good catch.

File: invest/templates/home/home_page.html (hunk 3 above)
Comment (stuaxo) [reply invest/templates/home/home_page.html]: I'm going to pass this data in during the next PR, so might leave it for now (but yeah, it's pretty unsightly)

File: config/settings.py (hunk 1 above)
Comment (stuaxo) [reply config/settings.py]: Just read the bit about need a different settings file, couldn't agree more.   It's not a battle I want to fight right now, but if we ever get past MVP + April the 1st I' definitely up for it.

File: home/models.py (hunk 2 above)
Comment (stuaxo) [reply home/models.py]: Just writing some lisp.      Must be bad, I'm getting a bit confused even trying to arrange these brackets sensibly.

Comment (stuaxo) [reply home/models.py]: https://plugins.jetbrains.com/plugin/10080-rainbow-brackets

Comment (stuaxo) [reply home/models.py]: https://www.emacswiki.org/emacs/RainbowDelimiters

Comment (sdonk) [reply home/models.py]: Do you use Pycharm? @stuaxo 

If yes then code -> reformat code

Comment (stuaxo) [reply home/models.py]: image

Comment (stuaxo) [reply home/models.py]: image

File: invest/templates/includes/how_we_help_cards.html (hunk 4 above)
Comment (stuaxo) [reply invest/templates/includes/how_we_help_cards.html]: Done

**Pull request merged**